main.js

```diff
@@ -1,12 +1,26 @@
 var car = {
-  direction: 'east'
+  direction: 'east',
+  xAxis: 0,
+  yAxis: 0
 };
 
 var carDOM = document.querySelector('.car');
 
 document.addEventListener('keydown', carFunction);
 
 function carFunction(event) {
+  if (event.key === ' ') {
+    if (car.direction === 'north') {
+      setInterval(function () { car.yAxis--; carDOM.style = 'top: ' + car.yAxis.toString() + 'px; left: ' + car.xAxis.toString() + 'px;'; }, 16);
+    } else if (car.direction === 'south') {
+      setInterval(function () { car.yAxis++; carDOM.style = 'top: ' + car.yAxis.toString() + 'px; left: ' + car.xAxis.toString() + 'px;'; }, 16);
+    } else if (car.direction === 'west') {
+      setInterval(function () { car.xAxis--; carDOM.style = 'top: ' + car.yAxis.toString() + 'px; left: ' + car.xAxis.toString() + 'px;'; }, 16);
+    } else if (car.direction === 'east') {
+      setInterval(function () { car.xAxis++; carDOM.style = 'top: ' + car.yAxis.toString() + 'px; left: ' + car.xAxis.toString() + 'px;'; }, 16);
```

Comment:
You should make these named functions rather than anonymous functions. It will also make your code easier to read.